labman/gui/static/js/plateViewer.js

```diff
@@ -263,33 +262,8 @@ PlateViewer.prototype.initialize = function (rows, cols) {
     var col = args.cell;
     var content = args.item[col];
 
-    if (that.plateId == null) {
-      // This is a new plate, we need to create the plate
-      var plateName = $('#newNameInput').val().trim();
-      var plateConf = $('#plate-conf-select option:selected').val();
-      $.post('/process/sample_plating', {'plate_name': plateName, 'plate_configuration': plateConf}, function (data) {
-        that.plateId = data['plate_id'];
-        that.processId = data['process_id'];
-
-        // we can only instantiate the notes box when we have a process id
-        that.notes = new NotesBox(that.container.parent(),
-                                  '/process/sample_plating/notes',
-                                  that.processId);
-
-        $('#plateName').prop('pm-data-plate-id', that.plateId);
-        $('#plateName').prop('pm-data-process-id', that.processId);
-        // Once the plate has been created, we can disable the plate config select
-        $('#plate-conf-select').prop('disabled', true);
-        // The plate has been created, plate the sample
-        that.modifyWell(row, col, content);
-      })
-        .fail(function (jqXHR, textStatus, errorThrown) {
-          bootstrapAlert(jqXHR.responseText, 'danger');
-        });
-    } else {
-      // The plate already exists, simply plate the sample
-      that.modifyWell(row, col, content);
-    }
+    // The plate already exists, simply plate the sample
+    that.modifyWell(row, col, content);
   });
 
   // When the user right-clicks on a cell
```

Comment:
Is it possible to make the well comment text field automatically scoped for entering text when it's displayed? As it stands now it requires an additional mouse click to activate.